rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -273,14 +341,14 @@ def get_service_node(sn_name):
                 return sn
 
     nodes = {}
+    status_getter = get_concurrent_status_checker()
     for service_node_name in remote_node_names:
         service_node = get_service_node(service_node_name)
-        creds = Credentials.get(service_type, service_node.private_ip)
-        prometheus_response = status_func(
-            'https://{ip}:53333/monitoring/'.format(
-                ip=service_node.private_ip),
-            auth=(creds.username, creds.password),
-            ca_path=creds.ca_path)
+        status_getter.append(
```

Comment:
the `.append` + `.get_responses` calling approach seems too complex, can't it just be `.get_responses(remote_node_names)` or similar?

Reply:
or of course `.get_responses(get_service_node(name) for name in remote_node_names)` or however you want to spell it :)